harness/determined/exec/gc_checkpoints.py

```diff
@@ -46,6 +57,7 @@ def main(argv: List[str]) -> None:
         action="version",
         version="Determined checkpoint GC, version {}".format(det.__version__),
     )
+    parser.add_argument("--experiment-id", help="The experiment ID to run the GC job for")
```

Comment:
It occurs to me that some day when we want to version the harness/master protocol, we'll have to either pack the gc script in at runtime or we will have to edit it several versions before we start using a new feature in it.

Just a thought, not related to this PR in particular.

Reply:
yeah.. that's a good point. interesting.